OpenShift/helm/nexus-repository-manager/templates/additional-service.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.service.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+{{- if .Values.service.name }}
+  name: {{ .Values.service.name }}
+{{- else }}
+  name: {{ template "nexus.name" . }}-service
+{{- end }}
+  labels:
+{{ include "nexus.labels" . | indent 4 }}
+{{- if .Values.service.labels }}
+{{ toYaml .Values.service.labels | indent 4 }}
+{{- end }}
+{{- if .Values.service.annotations }}
+  annotations:
+{{ toYaml .Values.service.annotations | indent 4 }}
+{{- end }}
+spec:
+  ports:
+  {{- if .Values.service.portName }}
+  - name: {{ .Values.service.portName }}
+    port: {{ .Values.service.port }}
+    targetPort: {{ .Values.service.targetPort }}
+  {{- end }}
+  {{- with .Values.service.ports  }}
+{{ toYaml . | indent 2 }}
+  {{- end }}
+  selector:
+    app: {{ template "nexus.name" . }}
+    release: {{ .Release.Name }}
+  type: {{ .Values.service.serviceType }}
+  {{ if .Values.service.loadBalancerSourceRanges }}
+  loadBalancerSourceRanges:
+    {{- range .Values.service.loadBalancerSourceRanges }}
+    - {{ . }}
+    {{- end }}
+  {{ end }}
+{{- end}}
+
```

Comment:
this is merely what used to be service.yaml.  it serves the "additional services" section of the values.yaml.

Reply:
not sure why we need  this

Reply:
this looks like it is doing what we were doing in `service.yaml`. What's the motivation behind purging that file and moving it here?

Reply:
I don't know what these really are either. 
@adrianpowell It is indeed what used to be in service.yaml, but I wanted service.yaml to match the IQ chart and to be the simple template to establish the ClusterIP. I renamed all this pre-existing stuff to additional-service.yaml, so I can keep it without knowing much about it. It's no less documented than it was previously. I can take a crack at learning it and documenting it, or I could easily remove it too.

Reply:
Sounds reasonable, thanks for the clarification

Reply:
to expound on my philosophy slightly, i regard these templates to have some levels:
* core functionality which we need and document
* extra functionality we don't use by default but understand enough to document
* extra functionality we don't understand, and don't document, but I'd sure hate to remove it for someone who does understand it.
This one falls in that area where i'd hate to take it away from someone who may be using it.

Reply:
I moved it to preserve the functionality in case someone wanted to use it, but still make room for the more standard/simplified service.yaml from IQ.